pages/profile/[ethAddress].tsx

```diff
@@ -0,0 +1,199 @@
+import { NextPage } from "next"
+import { useEffect, useRef, useState } from "react"
+
+// @ts-expect-error
+import Blockies from 'react-blockies'
+
+import Papa from 'papaparse'
+
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineController,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+  Filler,
+  ChartData
+} from 'chart.js'
+import { Chart } from 'react-chartjs-2'
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineController,
+  LineElement,
+  Filler,
+  Title,
+  Tooltip,
+  Legend
+)
+
+export function NationCredChart({ profile }: any) {
+  console.info('NationCredChart')
+
+  const chartRef = useRef<ChartJS>(null)
+  const [chartData, setChartData] = useState<ChartData<'line'>>({
+    datasets: [],
+  })
+
+  useEffect(() => {
+    console.info('useEffect')
+    const chart = chartRef.current
+
+    if (!chart) {
+      return
+    }
+
+    let colorGradient = chart.ctx.createLinearGradient(0, 0, 0, 400)
+    colorGradient.addColorStop(0, 'rgba(213, 163, 152, 0.2)')
+    colorGradient.addColorStop(1, 'rgba(213, 163, 152, 0.8)')
+
+    // Fetch data from datasets repo
+    const sourceCredFileUrl: string = `https://raw.githubusercontent.com/nation3/nationcred-datasets/f9e5b8d30b969815a918de619fa4e89a1c800ce3/data-sources/sourcecred/output/sourcecred-${profile.ethAddress}.csv`
```

Comment:
@johnmark13 Using the data from your PR here.  Once merged, we can update this URL.